app/assets/javascripts/host_edit.js

```diff
@@ -2,6 +2,56 @@ $(document).on('ContentLoad', function(){onHostEditLoad()});
 $(document).on('AddedClass', function(event, link){load_puppet_class_parameters(link)});
 $(document).on('click', '#params-tab', function() { resizeTextareas($('#params')); });
 
+LookupValueController = function(item) {
+  var self = this; //because JS' this confuses me
+  var element = item,
+  id = element.data("lookupKeyId"),
+  name = element.find(".lookup_key_name").html(),
+  value = element.find(".value").val(),
+  overrideTR = $("#lookup_value_for_"+id),
+  destroyer = overrideTR.find(".destroyer"),
+  hiddenToggle = overrideTR.find(".hidden_value input"),
+  overridingField = overrideTR.find(".overriding-value");
+
+  addEventListeners = function() {
+    element.find(".override_button").click(override);
+    overrideTR.find(".remove_override_button").click(removeOverride);
+    hiddenToggle.change(toggleHidden);
+  }
+
+  override = function() {
```

Comment:
This doesn't seem to add a strikethrough (override-param class) to the textareas, ditto in the removeOverride function below which doesn't seem to remove it.